doc/api/process.markdown

```diff
@@ -237,13 +240,13 @@ Note:
 - `SIGKILL` cannot have a listener installed, it will unconditionally terminate
   Node.js on all platforms.
 - `SIGSTOP` cannot have a listener installed.
-
-Note that Windows does not support sending Signals, but Node.js offers some
-emulation with `process.kill()`, and `child_process.kill()`:
-- Sending signal `0` can be used to search for the existence of a process
```

Comment:
I'm sure the text could use improvement, so if there is some trouble with it lets address that, but the statements are Windows specific.